packages/core/src/client.ts

```diff
@@ -2206,6 +2206,11 @@ export class MedplumClient extends EventTarget {
       return undefined as unknown as T;
     }
 
+    if (response.status === 404) {
+      console.error(`404, couldn't retrieve response from ${url}`);
```

Comment:
Other error messages are prepending with e.g. "Error: "; consider aligning with that pattern.  Maybe something like `Error: URL ${url} not found`

Reply:
Good call! I'll add Error:

Reply:
Since we're already returning an `OperationOutcomeError`, do we even need this logging statement?